src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -427,6 +443,12 @@ public function findHostGroups(): array
             $value = $data[$type];
             $statement->bindValue($key, $value, $type);
         }
+
+        if (!empty($hostId)){
```

Comment:
```suggestion
        if ($hostId !== null) {
```

Reply:
you can't have an entity with id = 0, at least there shouldn't be such, but ok let's apply it